app/javascript/mastodon/features/account/components/header.js

```diff
@@ -240,6 +241,7 @@ class Header extends ImmutablePureComponent {
       menu.push({ text: intl.formatMessage(messages.follow_requests), to: '/follow_requests' });
       menu.push({ text: intl.formatMessage(messages.favourites), to: '/favourites' });
       menu.push({ text: intl.formatMessage(messages.lists), to: '/lists' });
+      menu.push({ text: intl.formatMessage(messages.followed_tags), to: '/followed_tags' });
```

Comment:
Not really an issue with your PR, but I'm noticing just now that there are lots of inconsistencies between this dropdown and the composer one. I'm not sure it makes sense to replicate all of this in both and risk having them out of sync…

Reply:
Should we open an issue to extract this into its own component to reuse in both places?